src/cider/nrepl/middleware/test.clj

```diff
@@ -79,17 +80,18 @@
         c (when (seq test/*testing-contexts*) (test/testing-contexts-str))
         i (count (get-in (@current-report :results) [ns (:name (meta v))]))
         gen-input (:gen-input @current-report)
+        dd-pprint-str  #(with-out-str (dd/pretty-print %))
         pprint-str #(with-out-str (pp/pprint %))]
     ;; Errors outside assertions (faults) do not return an :expected value.
     ;; Type :fail returns :actual value. Type :error returns :error and :line.
     (merge (dissoc m :expected :actual)
            {:ns ns, :var (:name (meta v)), :index i, :context c}
            (when (and (#{:fail :error} t) (not fault))
-             {:expected (pprint-str expected)})
+             {:expected (dd-pprint-str expected)})
```

Comment:
Why is this changed unconditionally? Shouldn't you be using the methods from the new namespace here?

Reply:
Yes, you're right. Updated the PR.